### ISSUE TITLE
MANYBODY package which is not enabled in this LAMMPS binary

### ISSUE THREAD

**Issue by erny123** [lammps/lammps#2037]
**Summary**

Trying to run the following tutorial: 
[https://icme.hpc.msstate.edu/mediawiki/index.php/LAMMPS_Tutorial_1](https://icme.hpc.msstate.edu/mediawiki/index.php/LAMMPS_Tutorial_1)
When I run it using the following:
`mpirun -np 4 $LDir/lmp_mpi -in calc_fcc.in`

It fails and output shown below. In summary:
`ERROR: Unrecognized pair style 'eam/alloy' is part of the MANYBODY package which is not enabled in this LAMMPS binary. (../force.cpp:285)
Last command: pair_style eam/alloy`

**LAMMPS Version and Platform**

Not sure how to check what LAMMPS version I have but I just installed it 3 weeks ago. 
Running this on Windows 10 Linux Subsystem Ubuntu 18
6 Core AMD processor

As explained in the summary, trying to follow the tutorials from MS State, and running into some problems. I know the tutorial is pretty old but the fact that it says the MANYBODY package is not enabled concerns me. I can't find anything on the LAMMPS manual on how to enable a package. 

LAMMPS Input: 
`mpirun -np 4 $LDir/lmp_mpi -in calc_fcc.in`

LAMMPS Output: 

`LAMMPS (15 Apr 2020)
Lattice spacing in x,y,z = 4 4 4
Created orthogonal box = (0 0 0) to (4 4 4)
  1 by 2 by 2 MPI processor grid
Lattice spacing in x,y,z = 4 4 4
Created 4 atoms
  create_atoms CPU = 0.0006632 secs
Replicating atoms ...
  orthogonal box = (0 0 0) to (4 4 4)
  1 by 2 by 2 MPI processor grid
  4 atoms
  replicate CPU = 0.0009239 secs
ERROR: Unrecognized pair style 'eam/alloy' is part of the MANYBODY package which is not enabled in this LAMMPS binary. (../force.cpp:285)
Last command: pair_style eam/alloy

Primary job  terminated normally, but 1 process returned
a non-zero exit code.. Per user-direction, the job has been aborted.

mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[58459,1],0]
  Exit code:    1
`

**Comment by akohlmey** [comment]
> As explained in the summary, trying to follow the tutorials from MS State, and running into some problems. I know the tutorial is pretty old but the fact that it says the MANYBODY package is not enabled concerns me. I can't find anything on the LAMMPS manual on how to enable a package.

The LAMMPS manual has *VERY* detailed instructions on how to compile LAMMPS (the entire chapter 3) and how to include optional packages. Please see https://lammps.sandia.gov/doc/Build_package.html
Examples given there even explicitly show how to include the MANYBODY package.

**Comment by akohlmey** [comment]
I also find it surprising how you can compile three weeks ago a version that is 12 days old, unless you have discovered the secret of time travel. In that case, please share that knowledge.

**Comment by erny123** [comment]
akohmey,
I'm sorry for offending you. 
One thing isn't clear to me. What packages are included with the command: 
`cmake --build .`

Also, it says:
[https://lammps.sandia.gov/doc/Build_cmake.html](https://lammps.sandia.gov/doc/Build_cmake.html)
> Compilation can take a long time, since LAMMPS is a large project with many features. If your machine has multiple CPU cores (most do these days), you can speed this up by compiling sources in parallel with make -j N (with N being the maximum number of concurrently executed tasks).

Is 
`make -j N`
The same as the above cmake command? Can they be used interchangeably? 

Last, I installed and compiled LAMMPS from [https://lammps.sandia.gov/doc/Howto_bash.html#compiling-lammps-in-bash-on-windows](https://lammps.sandia.gov/doc/Howto_bash.html#compiling-lammps-in-bash-on-windows)
In order to use CMAKE I used 
`make no-all purge`
But this command does not work anywhere and the documentation does not say where to use this command so I used it in the /src folder. 

After following the steps in section 3 and 8 I get to the point with:
`cmake -C ../cmake/preset/minimal.cmake -D PKG_MSC=on ../cmake`
And I get an error:
`CMake Error: Error processing file: ../cmake/preset/minimal.cmake
-- Running check for auto-generated files from make-based build system
-- Found OpenMP_CXX: -fopenmp
-- Found OpenMP: TRUE
-- Could NOT find FFMPEG (missing: FFMPEG_EXECUTABLE)
-- Performing Test COMPILER_SUPPORTS-march=native
-- Performing Test COMPILER_SUPPORTS-march=native - Success
-- Generating style_angle.h...
-- Generating style_atom.h...
-- Generating style_body.h...
-- Generating style_bond.h...
-- Generating style_command.h...
-- Generating style_compute.h...
-- Generating style_dihedral.h...
-- Generating style_dump.h...
-- Generating style_fix.h...
-- Generating style_improper.h...
-- Generating style_integrate.h...
-- Generating style_kspace.h...
-- Generating style_minimize.h...
-- Generating style_nbin.h...
-- Generating style_npair.h...
-- Generating style_nstencil.h...
-- Generating style_ntopo.h...
-- Generating style_pair.h...
-- Generating style_reader.h...
-- Generating style_region.h...
-- Generating packages_angle.h...
-- Generating packages_atom.h...
-- Generating packages_body.h...
-- Generating packages_bond.h...
-- Generating packages_command.h...
-- Generating packages_compute.h...
-- Generating packages_dihedral.h...
-- Generating packages_dump.h...
-- Generating packages_fix.h...
-- Generating packages_improper.h...
-- Generating packages_integrate.h...
-- Generating packages_kspace.h...
-- Generating packages_minimize.h...
-- Generating packages_nbin.h...
-- Generating packages_npair.h...
-- Generating packages_nstencil.h...
-- Generating packages_ntopo.h...
-- Generating packages_pair.h...
-- Generating packages_reader.h...
-- Generating packages_region.h...
-- Generating lmpinstalledpkgs.h...
-- The following tools and libraries have been found and configured:
 * Git
 * MPI
 * OpenMP
 * JPEG
 * PNG
 * ZLIB`
